Source: written by Pierre Lejeune <superheron@gmail.com>
Upstream: fixed in git by removal of gtk+:2
Reason: Fix typo in configure.in
--- sound-juicer-2.32.0/configure.in.orig	2010-09-28 11:54:47.000000000 +0200
+++ sound-juicer-2.32.0/configure.in	2011-05-01 15:11:35.016918705 +0200
@@ -55,7 +55,7 @@
        GTK_REQUIRED=2.90.0
        LIBBRASERO_MEDIA_PC=libbrasero-media3
        LIBCANBERRA_GTK_PC=libcanberra-gtk3
-       GNOME_MEDIA_PROFILES_PC=gnome-media-profiles-3.0
+       GNOME_MEDIA_PROFILES_PC=libgnome-media-profiles-3.0
        ;;
 esac
 
